### PR TITLE
Apply feedback

### PR DESCRIPTION
**Pull request by andrewdes**
Addressing feedback in #23

### CODE REVIEW

**Comment by yeekangc** [comment]
Similar feedback that I provided for the IBM Cloud guide in the latest PR applies here for the Intro section.

Please work with @gkwan-ibm till you all are ready and I can take another look then.

**Comment by andrewdes** [comment]
I saw https://github.com/OpenLiberty/draft-guide-cloud-aws/issues/22 and thought we wanted to work EKS into the title and abstract so I made a similar change to IBM Cloud to keep the 2 cloud guides consistent. I will rework the title.

File: README.adoc [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
 :page-tags: ['Kubernetes', 'Docker']
 :page-permalink: /guides/{projectid}
-:page-related-guides: ['kubernetes-intro']
+:page-related-guides: ['kubernetes-intro', 'kubernetes-microprofile-config', 'kubernetes-microprofile-health', 'istio-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-:page-seo-title: Amazon Elastic Container Service (EKS) tutorial
-:page-seo-description: How to deploy microservices to Amazon EKS
+:page-seo-title: Deploying microservices to Amazon Elastic Container Service for Kubernetes (EKS)
+:page-seo-description: A tutorial on how to deploy microservices to Amazon EKS using ECR as a private Docker registry.
```
Comment (yeekangc) [inline README.adoc]: We should call (spell) out ECR.

File: README.adoc [hunk 2]
```diff
@@ -11,21 +11,21 @@
...
 :guide-author: Open Liberty
-= Deploying microservices to Amazon Elastic Container Service for Kubernetes
+= Deploying microservices to Amazon Web Services (AWS)
```
Comment (yeekangc) [inline README.adoc]: I will take out "(AWS)" here.

File: README.adoc [hunk 3]
```diff
@@ -11,21 +11,21 @@
...
 
 [.hidden]
 NOTE: This repository contains the guide documentation source. To view the guide in published form, view it on the https://openliberty.io/guides/{projectid}.html[Open Liberty website^].
- 
-Explore how to push Docker images to an Amazon Elastic Container Registry and run them inside of containers on Amazon Elastic Container Service for Kubernetes.
+
+Explore how to deploy microservices to Amazon Elastic Container Service for Kubernetes (Amazon EKS).
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
Explore how to deploy microservices to Amazon Elastic Container Service for Kubernetes (EKS) on Amazon Web Services (AWS).
```

File: README.adoc [hunk 4]
```diff
@@ -55,18 +43,32 @@ you to more easily scale your microservices and serve them in a high-availabilit
...
+
+Kubernetes is an open source container orchestrator that automates many tasks involved in 
+deploying, managing, and scaling containerized applications. If you would like to learn
+more about Kubernetes, check out the https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^]
+guide.
+
+Various clouds offer solutions for running your workloads in a {kube} cluster. Using a 
+cloud-based infrastructure allows you to focus on developing your microservices rather 
+than worrying about details that are related to the servers you deploy them to. Using 
+cloud allows you to easily scale your microservices and serve them in a high-availability
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
a cloud allows you to easily scale your microservices and serve them in a high-availability 
```

File: README.adoc [hunk 5]
```diff
@@ -55,18 +43,32 @@ you to more easily scale your microservices and serve them in a high-availabilit
...
+cloud-based infrastructure allows you to focus on developing your microservices rather 
+than worrying about details that are related to the servers you deploy them to. Using 
+cloud allows you to easily scale your microservices and serve them in a high-availability 
+setup.
+
+Amazon Web Services (AWS) offers a managed Kubernetes service called Amazon EKS. EKS 
+simplifies the process of running Kubernetes on AWS without needing to install or maintain 
+your own Kubernetes control plane. It provides a hosted {kube} cluster that you can deploy 
+your microservices to. You will use it in conjunction with Amazon Elastic Container 
+Registry (ECR).
```
Comment (yeekangc) [inline README.adoc]: Consider explaining what is ECR and why ... at an appropriate place.

**Pull request merged**